<launch>
	<node name="shuc_sound_player" pkg="shuc_audio" type="shuc_sound_player.py">
		<param name= "audio_files_path" value = "$(find shuc_audio)/audio/"/>
	</node>
	<node name="shuc_sound_selector" pkg="shuc_audio" type="shuc_sound_selector.py"/>
	<node name="sound_play" pkg="sound_play" type="soundplay_node.py"/>
</launch>
